doc/rados/operations/health-checks.rst

```diff
@@ -1153,22 +1157,22 @@ For more information, see :ref:`specifying_pool_target_size`.
 TOO_FEW_OSDS
 ____________
 
-The number of OSDs in the cluster is below the configurable
-threshold of ``osd_pool_default_size``.
+The number of OSDs in the cluster is below the configurable threshold of
+``osd_pool_default_size``.
```

Comment:
Suggest appending `This means that some or all data may not be able to satisfy the data protection policy specified in CRUSH rules and pool settings`.